internal/lsp/completions/providers/shared.go

```diff
@@ -0,0 +1,3 @@
+package providers
+
+const fileURI = "file:///foo/bar/file.rego"
```

Comment:
What does this do?

Reply:
I dislike this but https://golangci-lint.run/usage/linters/#goconst is on and there doesn't seem to be a way to ignore test files...

I have made it clearer in 166faa0da9c44324714a3bc28c2d3bcc5d611d80, but I'd rather just disable goconst if we can for such things. I'd rather have it repeated as I feel like it makes the tests clearer.